pkg/api/unversioned/types.go

```diff
@@ -302,13 +302,6 @@ const (
 	CauseTypeUnexpectedServerResponse CauseType = "UnexpectedServerResponse"
 )
 
-func (*ListOptions) IsAnAPIObject()     {}
```

Comment:
There was no point for them.

On Thu, Dec 10, 2015 at 7:53 AM, David Eads <notifications@github.com>
wrote:

> In pkg/api/unversioned/types.go
> <https://github.com/kubernetes/kubernetes/pull/18473#discussion_r47222232>
> :
>
> > @@ -302,13 +302,6 @@ const (
> >  	CauseTypeUnexpectedServerResponse CauseType = "UnexpectedServerResponse"
> >  )
> >
> > -func (*ListOptions) IsAnAPIObject()     {}
>
> What happened to these? No longer worthy?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/18473/files#r47222232>.
>